client/src/utils/constants/Languages/langItem.ts

```diff
@@ -1,5 +1,5 @@
-export const langItem = [ 
-    { item: 'ru-RU' },
-    { item: 'en-EN' },
-    { item: 'ua-UA' }
-];
\ No newline at end of file
+export const langItem = [
+    { item: 'ru-RU', label: 'Russian' },
```

Comment:
key named 'item' in array 'langItem', use names, that describes content of variable, keys etc

Reply:
Array shouldn't be named like item of array. use langList instead for example